Automattic/wp-calypso
# Licensing Portal: Add notices about automatic billing on credit card management screens

PT: pdpAdu-bj-p2
Card: 1201902532330136-as-1201935935452151

#### Changes proposed in this Pull Request

* Licensing Portal: Add notices that the primary payment method will be used for automatic invoice settlement on credit card management screens.

#### Testing instructions

* Apply patch and spin up Jetpack Cloud.
* Open http://jetpack.cloud.localhost:3000/partner-portal/payment-methods/add and confirm the notice is visible as in Screenshot 1. Complete the form to create a primary payment method.
* Open http://jetpack.cloud.localhost:3000/partner-portal/payment-methods/add again, uncheck "Set as primary payment method", and confirm the notice is not visible as in Screenshot 2. Complete the form to create a secondary payment method.
* Open http://jetpack.cloud.localhost:3000/partner-portal/payment-methods, click on the delete option for your primary payment method and confirm the notice is visible as in Screenshot 3. Complete the deletion.
* Open http://jetpack.cloud.localhost:3000/partner-portal/payment-methods, click on the delete option for your new primary payment method and confirm the notice is visible as in Screenshot 4. Complete the deletion to clean up after your tests.

Screenshot 1: Add primary payment method
![Screen Shot 2022-04-13 at 16 32 06](https://user-images.githubusercontent.com/22746396/163192467-b6474dc9-09b2-4cb2-8abc-7b813fbd939c.png)

Screenshot 2: Add a secondary payment method
![Screen Shot 2022-04-13 at 16 32 21](https://user-images.githubusercontent.com/22746396/163192473-7dbbbb57-f0f4-42ae-b4b2-7ec03fd4a075.png)

Screenshot 3: Delete the primary payment method when a secondary is available
![Screen Shot 2022-04-13 at 16 32 57](https://user-images.githubusercontent.com/22746396/163192477-d0c9ee72-286b-4033-b43f-a9ca92fd6e52.png)

Screenshot 4: Delete the primary payment method when a secondary is not available
![Screen Shot 2022-04-13 at 16 30 56](https://user-images.githubusercontent.com/22746396/163192489-541c76ce-55b4-4833-94d1-0e4190487f57.png)
